### PR TITLE
fix: handle dynamic js rendering #24

### PR DESCRIPTION
**Pull request by YegorZaremba**
fix: #24

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "Hide NSFW content from websites using this extension powered by AI! This extension replaces images that are NSFW with random images from Unsplash.",
   "scripts": {
     "copy": "cp src/content.js dist/src/",
-    "build": "parcel build src/background.js -d dist/src/ -o background && npm run copy",
```
Comment (navendu-pottekkat) [inline package.json:7]: Why was this changed?

**Comment by YegorZaremba** [comment]
Not enough
I'll merge all fixes into `dev`, than make PR with `dev` to `master`, which we should test a lot, and v1.0.0 is finished
you can follow status of work [here](https://github.com/navendu-pottekkat/nsfw-filter/projects/2)

**Comment by yxlin118** [comment]
I think when I tested this with https://scrolller.com/nsfw, scrolling back up still lead to the issue of unsafe content being visible again? This is only on my end?

File: package.json (hunk 1 above)
Comment (YegorZaremba) [reply package.json:7]: It just build all files in `src` folder, and you dont need to copy files from `src` to `dist/src`, I need cause I do some `import` from `utils.js` and I need to babelify some ES6 code like `import/export`

**Comment by YegorZaremba** [comment]
@yxlin118 please check with `dev` branch, it sound't be like that
`git pull && git checkout dev && npm i` and load `dist` folder to your browser

**Comment by yxlin118** [comment]
@YegorZaremba I tried the `dev` branch and it works fine without the lagging issue. I did a similar approach using MutationObserver but @navendu-pottekkat says it doesn't work on his end and it slows down the browser? How are you guys testing this?

**Comment by YegorZaremba** [comment]
@yxlin118 According to your PR, you filter every time, when sth is mutate, but we need check only images mutation, so probably its a reason why is was laggy
But I think my solution is not good enough too
